README.md

```diff
@@ -51,24 +48,47 @@ Navigate to the Debug tab within the [Google Developers Console][dev-console] to
    require('@google/cloud-debug');
    ```
 
-At this point you can deploy your application to AppEngine or to Google Compute Engine and the debug agent will activate with no further action. If you want to run your application locally, you can still use the cloud debug agent, with a few additional steps:
+## Running on Google Cloud Platform
+
+There are three different services that can host Node.js application to Google Cloud Platform.
+
+### Google App Engine Managed VMs
+
+If you are using [Google App Engine Managed VMs](https://cloud.google.com/appengine/docs/managed-vms/), you do not have to do any additional configuration.
+
+### Google Compute Engine
+
+Your VM instances need to be created with `cloud-platform` scope if created via [gcloud](https://cloud.google.com/sdk) or the 'Allow API access' checkbox selected if created via the [console](https://console.developers.google.com) (see screenshot).
+
+![GCE API](doc/images/gce.png?raw=true)
+
+If you already have VMs that were created without API access and do not wish to recreate it, you can follow the instructions for using a service account under [running elsewhere](#running-elsewhere).
+
+### Google Container Engine
+
+Container Engine nodes need to also be created with the `cloud-platform` scope, which is configurable during cluster creation. Alternatively, you can follow the instructions for using a service account under [running elsewhere](#running-elsewhere). It's recommended that you store the service account credentials as [Kubernetes Secret](http://kubernetes.io/v1.1/docs/user-guide/secrets.html).
+
+## Running elsewhere
+
+If your application is running outside of Google Cloud Platform, such as locally, on-premise, or on another cloud provider, you can still use Cloud Debug.
+
+1. You will need to specify the numeric project ID of your project. Your numeric project ID is visible in [Google Cloud Platform Project](https://console.developers.google.com/project?_ga=1.258049870.576536942.1443543237), it may be something like `1002377379474884`.
```

Comment:
Looks like the preferred wording here is console: see https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/171/files